modules/photo/src/layerModelBlending.cpp

```diff
@@ -0,0 +1,201 @@
+/*M///////////////////////////////////////////////////////////////////////////////////////
```

Comment:
It is better to use modern 3-line license header.

Reply:
yes,modern 3-line license header looks better.